README.md

```diff
@@ -35,6 +35,20 @@ AWS driver specific
 - AWS_FLAVOR (defaults to t2.medium) size of the virtual machine
 - AWS_MACHINE_USERNAME (defaults to Administrator) administrator account on the machine
 
+Openstack driver specific
+ - OPENSTACK_USERNAME Openstack username
+ - OPENSTACK_PASSWORD Openstack password
+ - OPENSTACK_AUTH_URL Openstack auth URL
+ - OPENSTACK_REGION (Defaults to 'RegionOne') Openstack region name
+ - OPENSTACK_IMAGE image if of the Openstack image
+ - OPENSTACK_FLAVOR (defaults to 3 (m1.medium)) type id of the virtual machine
+ - OPENSTACK_SECURITY_GROUPS (Defaults to default) security groups with
```

Comment:
"security groups with which starts instances separated by a comma" -> "security groups to apply to the instance, separated with commas"